core/models.py

```diff
@@ -97,3 +97,18 @@ class Card(models.Model):
 
     def __str__(self):
         return "%s - %s (%s)" % (self.user.username, self.card_number, self.comment)
+
+
+class NfcCard(models.Model):
```

Comment:
@nicolaei (og @matsasc)

Skal være litt kritisk her. Nå har vi i praksis to modeller (`NfcCard` og `Card`) som har samme oppgave: mappe kort (NFC i begge tilfeller) til modell og tilhørende data.

Ville det ikke vært bedre og utvidet `Card`-modellen med de dataene man har behov for (jf. `NfcCard`)? På den måten kan man benytte både kort som har UID eller faktisk UiO kort hvor man får hentet ut faktisk kortnummer.